client/src/components/content/PropertiesPanel/PropertiesPanelView/PropertiesPanelView.jsx

```diff
@@ -0,0 +1,80 @@
+import { is } from 'bpmn-js/lib/util/ModelUtil';
+import React from 'react';
+import PropertiesPanelApplicationDropdown from '../PropertiesPanelApplicationDropdown/PropertiesPanelApplicationDropdown';
+import PropertiesPanelTaskDropdown from '../PropertiesPanelTaskDropdown/PropertiesPanelTaskDropdown';
+import { Input, Tooltip, Typography } from 'antd';
+import { InfoCircleOutlined } from '@ant-design/icons';
+import styles from './PropertiesPanelView.module.css';
+import '../PropertiesPanel.css';
+
+const { Text } = Typography;
+
+/**
+ * @component
+ * @description Shows PropertiesPanel for one selected BPMN-Element.
+ */
+const PropertiesPanelView = (props) => {
+  return (
+    <div className='element-properties' key={props.element.id}>
+      <fieldset>
+        <Text className={`label-on-dark-background ${styles.text}`}>
+          {is(props.element, 'bpmn:Task')
+            ? 'Activity'
+            : is(props.element, 'bpmn:Event')
+            ? 'Event'
+            : is(props.element, 'bpmn:Gateway')
+            ? 'Gateway'
+            : ''}
+        </Text>
+      </fieldset>
+
+      <fieldset>
+        <Text className='label-on-dark-background'>ID: </Text>
+        <Text className='label-on-dark-background'>{props.element.id}</Text>
+      </fieldset>
```

Comment:
Question: should we really show the activity ID in the sidebar? In the final product I would definitely deny that, but do we really need it for testing at the moment?

Reply:
I agree but that shouldn't be part of the refactoring issue. If we remove it then it should be done with the redesign of the frontend.

Reply:
Okay - then I will do this in the css refactoring issue